<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditorModuleBrush</class>
 <widget class="QWidget" name="MapEditorModuleBrush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="m_pText">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="m_pBrushType">
     <item>
      <property name="text">
       <string>pen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fill</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="m_pCurrentTile">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="interactive">
      <bool>false</bool>
     </property>
     <property name="resizeAnchor">
      <enum>QGraphicsView::NoAnchor</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_pBrushType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
